index.ts

```diff
@@ -79,12 +79,13 @@ export function missingTranslationHandler() {
 })
 export class NgJhipsterModule {
     static forRoot(providedConfig: ModuleConfig): ModuleWithProviders {
-        ConfigHelper.setModuleConfigOptions(providedConfig);
```

Comment:
Oh ok for that we just need to use exported function instead of inline arrows. Ill fix this